floris/tools/visualization.py

```diff
@@ -25,126 +25,6 @@
 from floris.utilities import rotate_coordinates_rel_west, wind_delta
 
 
-def show_plots():
```

Comment:
This function allows users to avoid having to import pyplot only to run `plt.show()`. In basic use cases for the plotting methods in floris, the plotting functions create and work with the matplotlib data that they need, so from the outside (the user perspective) it's often not important which plotting library is ultimately being used. I suggest to keep this function to preserve this support.

```python
from floris import FlorisInterface
import floris.visualization as wakeviz

fi = FlorisInterface(...)
plane = fi.calculate_horizontal_plane(...)
wakeviz.visualize_calculate_plane(plane)

wakeviz.show_plots()
```

Reply:
That's true, @misi9170 and I were thinking it was against our normal expectations.  plt.show is a matplotlib operation, like saving a figure to png or adding to a pdf, it's just one of several options open to the user.  It seemed more simple to us to end FLORIS' responsibilities to the object after we return it.  I guess this point we might discuss a little more,

Reply:
Interesting point @rafmudaf, I hadn't thought of that. I guess I feel that most users will be importing matplotlib.pyplot anyway---in examples 02, 03, and 17 that Paul changed here, `matplotlib.pyplot as plt` was always imported; and I don't think we have any intention of supporting any other plotting library anyway?

To me, it might occasionally save one import line for users that just want to see a figure as generated from our tools with no change; but the cost is that it (to me) creates confusion, since I'm so used to needing to call `plt.show()` to see plots, that it's not immediately clear what this function would do (when I see it called, I assume it will do something more than just call `plt.show()`, because it's use case for saving the matplotlib import was not obvious to me).

Of course, there's no need for users to call this function if we do keep it---they can just go ahead and call `plt.show()`. If we want to keep it, I'd suggest:
- Changing the function name to `show()`, rather than `show_plots()`, so that it exactly mimics the underlying matplotlib call
- Add a comment to the version on visualization.py explaining why we are directly wrapping matplotlib's `show()` with our own
- Add a comment to at least one of the examples (possibly 02) where we call `wakeviz.show_plots()` explaining that users can optionally directly call `matplotlib.pyplot.show()` if they'd prefer

Reply:
This works for me @misi9170 if it works for you @rafmudaf , I can implement if you green light

Reply:
Done!